### PR TITLE
Cleanup CS1570 article

### LINKED ISSUE #25053: third bullet point seems incorrect :"you need to represent them with > or <."
The third bullet point seems incorrect :"you need to represent them with `> or <`."
I do use `< `and` >` characters, and they are reported as incorrect. 
Should they be encoded instead?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 077ab322-a36b-082a-a203-bd6a4213b56c
* Version Independent ID: b0bce499-5b03-56ab-a636-8e09323373be
* Content: [Compiler Warning (level 1) CS1570](https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs1570)
* Content Source: [docs/csharp/misc/cs1570.md](https://github.com/dotnet/docs/blob/main/docs/csharp/misc/cs1570.md)
* Product: **dotnet-csharp**
* Technology: **csharp-diagnostics**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes https://github.com/dotnet/docs/issues/25053

### CODE REVIEW

**Review by Youssef1313 — COMMENTED** [review]
The main fix here is enclosing `&gt;` and `&lt;` with backticks. Everything else is just a cleanup.

File: docs/csharp/misc/cs1570.md [hunk 1]
```diff
@@ -2,43 +2,35 @@
...
-// CS1570.cs  
-// compile with: /W:1  
-namespace ns  
-{  
-   // the following line generates CS1570  
-   /// <summary> returns true if < 5 </summary>  
-   // try this instead  
-   // /// <summary> returns true if <5 </summary>  
-  
-   public class MyClass
```
Comment (Youssef1313) [inline docs/csharp/misc/cs1570.md:37]: I moved the doc comment to a method since it says "returns".

File: docs/csharp/misc/cs1570.md [hunk 2]
```diff
@@ -2,43 +2,35 @@
...
-}  
+XML comment on 'construct' has badly formed XML — 'reason'
+
+When using [**DocumentationFile**](../language-reference/compiler-options/output.md#documentationfile), any comments in the source code must be in XML. Any error with your XML markup will generate CS1570. For example:
+
+- If you are passing a string to a **cref**, such as in an [\<exception>](../programming-guide/xmldoc/exception.md) tag, the string must be enclosed in double quotation marks.
+
+- If you are using a tag, such as [\<seealso>](../programming-guide/xmldoc/seealso.md), which does not have a closing tag, you must specify a forward slash before the closing angle bracket.
+
+- If you need to use a greater-than or less-than symbol in the text of description, you need to represent them with `&gt;` or `&lt;`. Alternatively, you can use CDATA.
```
Comment (Youssef1313) [inline docs/csharp/misc/cs1570.md:20]: I mentioned CDATA here as an alternative. It's a good option, especially if there are lots of special symbols that needs to be escaped (commonly used with `<code>`)

**Review by BillWagner — APPROVED** [review]
Thanks @Youssef1313 

I've reviewed the changes, and I'll :shipit: now.

**Pull request merged**